docs/advanced/tls_porting.md

```diff
@@ -78,7 +100,7 @@ To enable the NV seed entropy source, you have to add `MBEDTLS_ENTROPY_NV_SEED`
 "macros": ["MBEDTLS_ENTROPY_NV_SEED", etc.],
 ```
 
-This makes sure the entropy pool knows it can use the NV seed entropy source. 
+This makes sure the entropy pool knows it can use the NV seed entropy source.
 
 You can read more about how to add a macro for your target [here](../mbed_OS/Targets.md).
```

Comment:
Could you please modify this link? that document no longer exists but it seems the contents were moved to https://github.com/ARMmbed/Handbook/blob/master/docs/advanced/mbed_targets.md